### PR TITLE
Nathanfishcircleci patch 2 eol launch agent

### PR DESCRIPTION
**Pull request by nathanfishcircleci**
# Description
Improve messaging for launch agent deprecation

# Reasons
We continue to have customers install launch agent for cloud. 

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [x] Break up walls of text by adding paragraph breaks.
- [x] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [x] Keep the title between 20 and 70 characters.
- [x] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [x] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [x] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [x] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/runner-installation-docker.adoc [hunk 1]
```diff
@@ -12,17 +12,17 @@ contentTags:
 :experimental:
 :machine:
 
-This page describes how to install CircleCI's machine runner with the Docker executor. If you are looking to set up self-hosted runners in a private Kubernetes cluster, please visit the <<container-runner#,Container runner>> page.
+This page describes how to install CircleCI's machine runner with the Docker executor on server. If you are looking to set up self-hosted runners in a private Kubernetes cluster, please visit the <<container-runner#,Container runner>> page.
 
 [#new-recommended-method-container-runner]
-== New recommended method (container runner)
+== New supported method for Docker Installation (container runner)
```
Comment (rosieyohannan) [inline jekyll/_cci2/runner-installation-docker.adoc:18]:
```suggestion
== Supported method for Docker installation (container runner)
```

File: jekyll/_cci2/runner-installation-docker.adoc [hunk 2]
```diff
@@ -12,17 +12,17 @@ contentTags:
...
 
 The <<container-runner#,container runner>> is the recommended approach for running containerized jobs on self-hosted runners. Container runner offers the ability to seamlessly define, publish, and use custom Docker images during job execution, as well as the ability to easily manage dependencies or libraries through custom Docker images instead of enumerating dependencies as part of `steps` in the `.circleci/config.yml` file.
 
 [#machine-approach-with-docker]
 == Machine-based approach with Docker
 
-If you cannot use the container runner linked above, you can use Docker and install machine runners with the instructions below.
+NOTICE: The instructions below are no longer supported for Cloud customers.
```
Comment (rosieyohannan) [inline jekyll/_cci2/runner-installation-docker.adoc:25]:
```suggestion
CAUTION: The instructions below are not supported for Cloud customers. 
```

File: jekyll/_cci2/runner-installation-linux.adoc [hunk 3]
```diff
@@ -13,7 +13,7 @@ contentTags:
 :machine:
 :linux:
 
-This page describes how to install CircleCI's machine runner on Linux.
+This page describes how to install CircleCI's machine runner on Linux for Server customers. Please note that this method is deprecated for cloud customers and see https://circleci.com/docs/install-machine-runner-3-on-linux/ for recommended Runner installation on Linux for cloud.
```
Comment (rosieyohannan) [inline jekyll/_cci2/runner-installation-linux.adoc:16]:
```suggestion
CAUTION: This installation method is deprecated for **CIrcleCI cloud** customers. Refer to the xref:install-machine-runner-3-on-linux#[machine runner 3.0 installation guide] for the recommended steps to set up Linux self-hosted runners on CircleCI cloud.
 
This page describes how to install CircleCI's machine runner on Linux for **CircleCI server** customers. 
```

File: jekyll/_cci2/runner-installation-mac.adoc [hunk 4]
```diff
@@ -13,7 +13,7 @@ contentTags:
 :machine:
 :macos:
 
-This page describes how to install CircleCI's machine runner on macOS.
+This page describes how to install CircleCI's machine runner on macOS. Please note that this method is deprecated for cloud customers and see https://circleci.com/docs/install-machine-runner-3-on-macos/ for recommended approach.
```
Comment (rosieyohannan) [inline jekyll/_cci2/runner-installation-mac.adoc:16]: @nathanfishcircleci are the steps on this page still correct for server? If so I can add the same wording as used in the linux page in this PR

File: jekyll/_cci2/runner-installation-windows.adoc [hunk 5]
```diff
@@ -13,7 +13,9 @@ contentTags:
 :machine:
 :windows:
 
-This page describes how to install CircleCI's machine runner on Windows. This has been tested for Windows Server 2019 and Windows Server 2016, both in Datacenter Edition. Other Server SKUs with Desktop Experience and Remote Desktop Services should also work.
+This page describes how to install CircleCI's machine runner on Windows for server. This has been tested for Windows Server 2019 and Windows Server 2016, both in Datacenter Edition. Other Server SKUs with Desktop Experience and Remote Desktop Services should also work.
+
+Please note that this method is deprecated for cloud customers and see https://circleci.com/docs/install-machine-runner-3-on-windows/ for recommended Runner installation on Windows for cloud.
```
Comment (rosieyohannan) [inline jekyll/_cci2/runner-installation-windows.adoc:18]:
```suggestion
CAUTION: This installation method is deprecated for **CIrcleCI cloud** customers. Refer to the xref:install-machine-runner-3-on-windows#[machine runner 3.0 installation guide] for the recommended steps to set up Windows self-hosted runners on CircleCI cloud.

This page describes how to install machine runner on Windows for **CircleCI server**. This has been tested for Windows Server 2019 and Windows Server 2016, both in Datacenter Edition. Other Server SKUs with Desktop Experience and Remote Desktop Services should also work.
```

File: jekyll/_cci2/runner-installation-mac.adoc (hunk 4 above)
Comment (nathanfishcircleci) [reply jekyll/_cci2/runner-installation-mac.adoc:16]: Still good for server.

**Comment by nathanfishcircleci** [comment]
@rosieyohannan accepted your suggestions. Thank you.

**Review by rosieyohannan — APPROVED** [review]

**Pull request merged**